Source/JavaScriptCore/heap/MarkedSpaceInlines.h

```diff
@@ -89,18 +89,19 @@ inline Ref<SharedTask<void(Visitor&)>> MarkedSpace::forEachWeakInParallel()
         {
         }
 
-        void drain(Vector<WeakBlock*, batchSize>& results)
+        size_t drain(std::array<WeakBlock*, batchSize>& results)
```

Comment:
Could also return a `span`, which would make it easier to use a range-based for loop at the call site, although the pointer to the array would be redundant.